src/script/conversation/MessageRepository.ts

```diff
@@ -1187,7 +1184,7 @@ export class MessageRepository {
     return undefined;
   }
 
-  private async updateMessageAsFailed(conversationEntity: Conversation, eventId: string, error: BackendError) {
+  private async updateMessageAsFailed(conversationEntity: Conversation, eventId: string, error: BackendError | any) {
```

Comment:
Can't we go with just `unknown` and check what type of error it is inside the method?

Reply:
like this?
```
if (isBackendError(error) && error.label === BackendErrorLabel.FEDERATION_REMOTE_ERROR) 
```

Reply:
yes